frontend/public/components/deployment-config.tsx

```diff
@@ -40,66 +41,98 @@ const rolloutAction = (kind, obj) => ({
 const {ModifyCount, EditEnvironment, common} = Cog.factory;
 
 const menuActions = [
-  rolloutAction,
-  ModifyCount,
-  EditEnvironment,
-  ...common,
+  rolloutAction as any,
```

Comment:
What TypeScript error is this fixing?

Reply:
Line 90:
``` jsx
  <ActionsMenu actions={menuActions.map(a => a(kindObj, dc))} />
```
`Cannot invoke an expression whose type lacks a call signature. Type '((kind: any, obj: any) => { label: string; callback: () => Promise<any>; }) | CogAction' has no compatible call signatures.`

Reply:
Wondering if this works

```ts
const menuActions: CogAction[] = ...
````

Reply:
That does work, thanks!